_includes/header.html

```diff
@@ -20,7 +20,7 @@
       <ul class="nav navbar-nav">
         {% for item in site.data.navigation %}
           {% if item.subnav %}
-            <li class="dropdown {% if page.url contains item.url %}active{% endif %}">
+            <li class="dropdown {% if page.url contains item.match %}active{% endif %}">
```

Comment:
What if we make `match` optional and fall back to `url` when it's not provided. Something like this (untested, might need some tweaking):

```
{% assign match = item.match %}
{% if match == blank %}
  {% assign match = item.url %}
{% endif %}
...
<li class="dropdown {% if page.url contains match %}active{% endif %}">
```